packages/common/__tests__/barcode.test.ts

```diff
@@ -28,7 +28,7 @@ describe('createBarCode', () => {
           input: t[1],
           width: 10, // mm
           height: 10, // mm
-          backgroundcolor: '00000000', // 背景色を指定しないとjsQRでうまく解析できない
+          backgroundColor: '00000000', // 背景色を指定しないとjsQRでうまく解析できない
```

Comment:
While it's not directly related to the content of this PR, I've changed it to camel case.